libs/designsystem/src/lib/components/modal/modal-wrapper/modal-wrapper.component.spec.ts

```diff
@@ -25,12 +70,70 @@ describe('ModalWrapperComponent', () => {
     spectator.fixture.destroy();
   });
 
-  describe('title', () => {
+  describe("when 'collapseTitle' is enabled", () => {
+    /* 
+      Whether the title is displayed & truncated is not tested. 
+      This is the responsibility of the ionic components; we assume
+      they're working as intended. 
+
+      If needed, it should be implemented as an integration test. 
+    */
+
+    let ionContentElement: HTMLIonContentElement;
+    let ionTitleElement: HTMLIonTitleElement;
+    const testTitle = 'This is a long test title';
+
+    beforeEach(() => {
+      spectator = modalWrapperTestBuilder
+        .flavor('modal')
+        .collapsibleTitle(true)
+        .title(testTitle)
+        .component(TitleEmbeddedComponent)
+        .build();
+
+      ionContentElement = spectator.query('ion-content');
+      ionTitleElement = spectator.query('ion-title');
+    });
+
+    afterEach(() => {
+      spectator.fixture.destroy();
+    });
+
+    it('should not have any padding between content & toolbar', () => {
+      const ionContentToolbarElement: HTMLIonToolbarElement = ionContentElement.querySelector(
+        'ion-toolbar'
+      );
+      expect(ionContentToolbarElement).not.toBeUndefined();
+
+      expect(ionContentToolbarElement).toHaveComputedStyle({
+        'padding-top': '0px',
+        '--padding-top': '0px',
+        '--padding-bottom': '0px',
+        '--padding-start': '0px',
+        '--padding-end': '0px',
+      });
+      expect(ionContentElement).toHaveComputedStyle({ '--padding-top': '0px' });
+    });
+
+    it('should place the title in both the content & the header', () => {
+      const contentTitle = ionContentElement.querySelector('kirby-page-title').innerHTML;
+      const headerTitle = ionTitleElement.querySelector('kirby-page-title').innerHTML;
+
+      expect(contentTitle).toBe(testTitle);
+      expect(headerTitle).toBe(testTitle);
+    });
+  });
+
+  describe('with slotted kirby-page-title', () => {
+    let ionTitle: HTMLIonTitleElement;
+
     beforeEach(() => {
       spectator = modalWrapperTestBuilder
         .title('Test title')
+        .component(TitleEmbeddedComponent)
```

Comment:
This is because we're deprecating the `ModalConfig.title` field. Before we could rely on that field to cause a title to be rendered. Now we need a component with a `kirby-page-title` instead which `TitleEmbeddedComponent` contains 😄